scripts/install_platform.sh

```diff
@@ -14,6 +14,7 @@ stage "Install ostis-web-platform"
 "${PLATFORM_PATH}/scripts/install_submodules.sh"
 "${PLATFORM_PATH}/scripts/install_dependencies.sh" --dev
 "${PLATFORM_PATH}/scripts/build_sc_machine.sh"
+"${PLATFORM_PATH}/scripts/build_sc_component_manager.sh"
```

Comment:
Is it not script to platform without manager of components?

Reply:
It is is not is it, yes